src/bitops.c

```diff
@@ -574,11 +574,12 @@ void setbitCommand(client *c) {
     /* Return original value. */
     addReply(c, bitval ? shared.cone : shared.czero);
 }
+#define LL_STR_BUF_SIZE    32
```

Comment:
should we use `LONG_STR_SIZE` which we already have?
it says it's for long and not long long, but it seems sufficient, maybe we should update it's doc.

Reply:
on the other hand, there's a ton of other places calling ll2string and using `sizeof(buf)`, so why bother fixing this one?

Reply:
Hi Oran,
I was working on Bitfeilds so oberved this and thought it would be good.
LONG_STR_SIZE is available but the value is 21, here we need 32. I thought same to use LONG_STR_SIZE since the value is different so created new.

Reply:
why do we need 32? AFAICT long long can fit into 21.

frankly, i don't see any reason fixing this one, as we have a ton of others doing the same thing, and as long as they're using sizeof, that's probably safe.